backend/routes/restaurant.js

```diff
@@ -25,6 +26,39 @@ router.get('/', (req, res) => {
   );
 });
 
+// Get Specific Restaurant opening hours
+router.get('/openhours', (req, res) => {
+  const input = req.query;
+  if (!input.restaurantID) {
+    res.status(400).json({ error: '/restaurant/openhours GET endpoint needs a restaurantID query param' });
+    return;
+  }
+
+  connection.query(
+    `SELECT restaurant_db.HOURS.DayOfWeek, restaurant_db.HOURS.OpenTime,restaurant_db.HOURS.CloseTime 
```

Comment:
Could this be replaced with ```SELECT DayOfWeek,OpenTime,CloseTime FROM restaurant_db.HOURS WHERE restaurant_db.HOURS.RestaurantID = ${req.query.restaurantID};``` to remove the table join?

Reply:
I think we're checking the RESTAURANT table too. Could do with:

Reply:
I think because HOURS.RestaurantID is a foreignkey, it has to exists in restaurant_db.RESTAURANT or it can't be added so can use jacobs simplified version. I think I initially wrote it that way to be able to handle an input of Restaurant name.

Reply:
Oh okay sweet as. Whoops that makes sense :)

Reply:
do we need to specify 'restaurant_db' when selecting from HOURS and RESTAURANT ?

Reply:
Also, while changing what, please use a ? placeholder to prevent injections.

Reply:
> do we need to specify 'restaurant_db' when selecting from HOURS and RESTAURANT ?

Nope, removed that now.


> Could this be replaced with `SELECT DayOfWeek,OpenTime,CloseTime FROM restaurant_db.HOURS WHERE restaurant_db.HOURS.RestaurantID = ${req.query.restaurantID};` to remove the table join and make the query a bit clearer?

Done. also replaced with ? place holders.

Resolving.